learning/books.md

```diff
@@ -17,7 +17,7 @@
 * Stanley H. Chan. [Intro to Probability for Data Science](https://www.amazon.com/Introduction-Probability-Data-Science-Stanley/dp/1607857464/), November 2021. The book is also available freely as HTML and PDF at [probability4datascience.com](https://probability4datascience.com/index.html). Code is in Julia/Python/R/Matlab.
 * Sambit Kumar Dash. [Hands-on Julia Programming](https://in.bpbonline.com/products/hands-on-julia-programming), Bpb Publications, October 2021.
 * Erik Engheim. [Julia for Beginners](https://leanpub.com/julia-for-beginners), January 2021.
-* Lee Phillips. [Practical Julia](https://lee-phillips.org/juliabook/). “The secret weapon for scientists.” In final production; coming from No Starch Press in September 2023.
+* Lee Phillips. [Practical Julia](https://lee-phillips.org/juliabook/). “A Hands-On Introduction for Scientific Minds.” Published by No Starch Press; available from Amazon and all major booksellers.
```

Comment:
```suggestion
* Lee Phillips. [Practical Julia: A Hands-On Introduction for Scientific Minds](https://nostarch.com/practical-julia), No Starch Press, October 2023.
```
Formatting consistent with other entries (incl. direct link to where to get the book)